### ISSUE TITLE
Issues with CREW MACROS after update to Alteryx 11.8

### ISSUE THREAD

**Issue by daka001** [AdRiley/ChaosReignsWithin-Alteryx-Macros#23]
Hi!

Cant get CREW Macros to run after update.
Conditional runner (and Runner) Always get no run...

Error - Alteryx Engine: The Feature "API" is not licensed.

Something to do with new format of License key and AlteryxRunner.exe?

(also posted in Alteryx Community https://community.alteryx.com/t5/Advanced-Analytics/Issues-with-CREW-MACROS-after-update-to-Alteryx-11-8-3-40692/m-p/134640#M3683 )

After installing 11.7 again problem was gone.

/Daniel

**Comment by TimGarstin** [comment]
I am having the same issue. In 11.7 the conditional runner and runner macros work and not in 11.8

 I have got a new licence for 11.8 and not 11.7 which means I cannot run any of my workflows. 

Best,

Tim

**Comment by daka001** [comment]
I have installed 2018.1 and they work fine

**Comment by AdRiley** [comment]
Yes this was related to the licensing changes and is fixed in 2018.1.

**Comment by datamonk42** [comment]
Was the issue re-introduced with version 2018.3.5 by any chance? Mine were working fine up until I updated from 2018.3.4 to 2018.3.5. AlteryxRunner.exe is now returning error code -1073741515 when I use the List Runner tool in a workflow, and if I just try to execute the AlteryxRunner.exe directly I get a system error indicating AlteryxEngine.dll is missing from my computer. I've uninstalled/reinstalled without any change.
